event-land/scss/_typography.scss

```diff
@@ -0,0 +1,10 @@
+/*------------------------------------*\
+  #FORNT
+\*------------------------------------*/
+
+// Import Inter font from fonts.googleapis.com 
```

Comment:
I think this comment is redundant because the code is already explains it'll import fonts from Google fonts website. I recommend to add this to the Readme file in the License section.